embulk-input-jdbc/src/main/java/org/embulk/input/jdbc/JdbcInputConnection.java

```diff
@@ -386,4 +387,13 @@ public boolean tableExists(String tableName) throws SQLException
             return columnNamesBuilder.build();
         }
     }
+
+    public void showDriverVersion(){
+        try {
+            DatabaseMetaData meta = connection.getMetaData();
+            logger.info(String.format(Locale.ENGLISH,"Using JDBC Driver %s",meta.getDriverVersion()));
+        } catch( SQLException e ) {
```

Comment:
I think we don't need to catch SQLException because Connection#getMetaData#getDriverVersion doesn't throw an exception normally.
If thrown, we can't access database after all.